compliance/sparql/src/test/java/org/eclipse/rdf4j/query/parser/sparql/function/CustomAggregateFunctionEvaluationTest.java

```diff
@@ -175,6 +262,7 @@ private static void addData() throws IOException {
 					+ "    <http://example/book5> <urn:n>  \"31.11241cawda3\"^^xsd:string .\n"
 					+ "    <http://example/book6> <urn:n>  0.090000200001 .\n"
 					+ "    <http://example/book7> <urn:n>  31.3 .\n"
+					+ "    <http://example/book7> <urn:n>  29.243564 .\n"
```

Comment:
If you move these two statements up to the beginning of the string then the test with SUM and HAVING passes.